gcloud-java-core/src/main/java/com/google/gcloud/PageImpl.java

```diff
@@ -38,6 +40,38 @@
     Page<T> nextPage();
   }
 
+  /**
+   * Base class for page fetchers.
+   *
+   * @param <T> the value type that the page holds. {@code T} should implement
+   *     {@code java.io.Serializable}.
+   * @param <ServiceOptionsT> the service options type.
+   */
+  @SuppressWarnings("rawtypes")
```

Comment:
I might be wrong so please correct me but if I remove the annotation (also from `ServiceFactory`) and compile with java 7 I don't get any warning.